packages/explorer/README.md

```diff
@@ -1,32 +1,54 @@
 # World Explorer
 
 World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+For the full information about it, [see the docs](http://mud.dev/world-explorer).
 
-## Getting started
+## Installation
 
-1. **Install the package**
+Starting with MUD 2.2, the MUD templates come with the World Explorer installed. See [Getting started](https://mud.dev/quickstart) guide for MUD for more information. These instructions are how to use World Explorer on earlier versions of MUD.
 
-   ```sh
-   pnpm add @latticexyz/explorer
-   ```
+The easiest way to get World Explorer for earlier MUD versions is to create a project with the new template.
```

Comment:
this doesn't make sense to me

we first say "latest mud templates comes with explorer" and then "add to an earlier mud project by creating a new template"?

the only changes needed are adding explorer/indexer packages + adding to mprocs, right?

Reply:
should we even focus on earlier MUD versions here?

Reply:
maybe not for the readme but would be good for the changelog